packages/api/src/index.js

```diff
@@ -1,17 +1,13 @@
-const fetch = require('node-fetch');
+require('isomorphic-fetch');
 const fetchHar = require('fetch-har');
 const Oas = require('oas').default;
 const oasToHar = require('@readme/oas-to-har');
 const pkg = require('../package.json');
+const { FormDataEncoder } = require('form-data-encoder');
 
 const Cache = require('./cache');
 const { parseResponse, prepareAuth, prepareParams, prepareServer } = require('./lib');
 
-global.fetch = fetch;
-global.Request = fetch.Request;
-global.Headers = fetch.Headers;
-global.FormData = require('form-data');
```

Comment:
We don't need this line anymore because `fetch-har` automatically polyfills `FormData` out for us with the (spec-compliant) [formdata-node](https://npm.im/formdata-node) package now if it's missing. https://github.com/readmeio/fetch-har/blob/main/index.js#L26-L35